charts/wharf-helm/values.yaml

```diff
@@ -101,6 +101,33 @@ web:
   # listens on. [Read more (kubernetes.io/docs)](https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service)
   containerPort: 8080
 
+  # -- Settings for the OpenId Connect authentication login.
+  oidc:
+
+    # -- The url to the Security Token Service (STS). The authority issues tokens. This field is required.
+    authority: "https://login.microsoftonline.com/841df554-ef9d-48b1-bc6e-44cf8543a8fc/v2.0/.well-known/openid-configuration"
+
+    # -- The Client MUST validate that the aud (audience) Claim contains its client_id value registered at the Issuer identified by the iss (issuer) Claim as an audience. The ID Token MUST be rejected if the ID Token does not list the Client as a valid audience, or if it contains additional audiences not trusted by the Client.
+    clientId: "01fcb3dc-7a2b-4b1c-a7d6-d7033089c779"
+
+    # -- List of scopes which are requested from the server from this client. This must match the Security Token Service configuration for the client you use. The openid scope is required. The offline_access scope can be requested when using refresh tokens but this is optional and some Security Token Service do not support this or recommend not requesting this even when using refresh tokens in the browser.
+    scope: "openid profile email offline_access api://wharf-internal/read api://wharf-internal/admin api://wharf-internal/deploy"
+
+    # -- 0, 1, 2 can be used to set the log level displayed in the console.
+    logLevel: "debug"
+
+    # -- When set to true, refresh tokens are used to renew the user session. When set to false, standard silent renew is used. Default value is false.
+    useRefreshToken: true
```

Comment:
nitnit: Here it says
> Default value is false.

but on `ignoreNonceAfterRefresh` below it says

> Default is `false`.

Suggest sticking to one.
I personally prefer the second one.

Reply:
No need to specify the default. The `helm-docs` uses the value from `values.yaml` (this file) as the default values for each field.

Writing `Default value is false.` here adds duplicate "default declarations" in the README.md

Reply:
Ah I see. 👍🏻

Reply:
Removed comments altogether, and instead only refer to their documentation over at https://nice-hill-002425310.azurestaticapps.net/docs/documentation/configuration#config-values